### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ cd audit/log
 
 and then run example with:
 ```
-mvn clean && mvn compile
+mvn clean && mvn compile \\
```
Comment (dbenders) [inline examples/README.md:17]: is this correct?

**Comment by dbenders** [comment]
Don't you need to change the algorithm name here?

File: examples/README.md (hunk 1 above)
Comment (pangea-andrest) [reply examples/README.md]: No. Fixed

**Comment by pangea-andrest** [comment]
@dbenders fixed algorithm name also

**Review by dbenders — APPROVED** [review]

**Pull request merged**